src/quicktype-core/language/CPlusPlus.ts

```diff
@@ -335,6 +335,160 @@ export type TypeContext = {
     inJsonNamespace: boolean;
 };
 
+interface StringType
+{
+    getType(): string;
+    getConstType(): string;
+    getSMatch(): string;
+    getRegex(): string;
+    createStringLiteral(inner: Sourcelike[]): Sourcelike[];
```

Comment:
Just a nitpick - I'm fine with the code as-is: `Sourcelike[]` is itself a `Sourcelike`, so usually there's little reason to use the type `Sourcelike[]`.  The only case where I use it is when I need to do array manipulation with it.